panel/panel.py

```diff
@@ -136,64 +119,51 @@ def run_update_panel_cmd(panel, text=None):
     panel.run_command(cmd, {'text': text, 'clear_sel': clear_sel})
 
 
-def format_row(lineno, error_type, dic):
-    lineno = int(lineno) + 1
-    start = dic['start'] + 1
-    msg = dic['msg'].rstrip()
-    tmpl = " {LINENO:>5}:{START:<4} {ERR_TYPE:7} {linter:>12}: {code:12} {MSG}"
-    return tmpl.format(
-        LINENO=lineno, START=start, ERR_TYPE=error_type, MSG=msg, **dic)
+def format_row(item):
+    line = int(item["line"]) + 1
```

Comment:
I actually cast to int in Linter. Did you ever had an error here?

Reply:
No, that casting was not initially introduced by me. I just stuck with it.